src/api/db/models/Image.js

```diff
@@ -643,6 +690,16 @@ export default class AuthedImageModel {
     return await ImageModel.getLabels(projId);
   }
 
+  async deleteImage(input, context) {
+    if (!hasRole(this.user, WRITE_OBJECTS_ROLES)) throw new ForbiddenError;
+    return await ImageModel.deleteImage(input, context);
+  }
+
+  async deleteImages(input, context) {
+    if (!hasRole(this.user, WRITE_OBJECTS_ROLES)) throw new ForbiddenError;
+    return await ImageModel.deleteImages(input, context);
+  }
+
```

Comment:
I think we need a new role for deleting images, and only managers should be allowed to do it.

Also, why not just keep it to one function (`deleteImages`) that accepts an array of IDs rather than two functions? I'm imagining the client could just pass in an array with one image ID if it only wants to delete a single image. No strong feelings if there's a reason to have both functions.

Reply:
Just read some more code and I can see the value in splitting them out and b/c it's nice being able to call `deleteImage` internally, but I'm not sure we need to expose it to clients? Not sure it matters one way or another either. I don't have strong feelings.

Reply:
- New Delete Objects role has been created
- The `deleteImage` external API has been removed in favour of the single `deleteImages`